test/test_distributions.py

```diff
@@ -4398,7 +4399,7 @@ def f(sample, *values):
             values, sample = self._perturb(Dist, keys, values, sample)
             expected = f(sample, *values)
             actual = traced_f(sample, *values)
-            self.assertEqual(expected, actual,
+            self.assertEqual(expected, actual, allow_inf=True,
```

Comment:
What's up here?

Reply:
There is one elem is nan in both the expected and actual results (https://github.com/pytorch/pytorch/pull/14600#issuecomment-443105287).